cas-server-core-authentication/build.gradle

```diff
@@ -10,5 +10,9 @@ dependencies {
     compile libraries.spring
     compile libraries.hibernate
     testCompile project(':cas-server-core-web')
+
+    provided(libraries.springsecurity) {
+        transitive = false
```

Comment:
This probably should go into the dependency definition directly if possible. Ideally, you should just have: 
`provided libraries.springsecurity`.